psydac/ddm/partition.py

```diff
@@ -3,10 +3,10 @@
 
 from sympy.ntheory import factorint
 
-__all__ = ['mpi_compute_dims']
+__all__ = ['mpi_compute_dims', 'openmp_compute_dims']
 
 #==============================================================================
-def mpi_compute_dims( nnodes, gridsizes, min_blocksizes=None ):
+def mpi_compute_dims(nnodes, gridsizes, min_blocksizes=None):
```

Comment:
It seems to me that all the changes to this file are done with the sole purpose of printing a different error message depending on whether MPI or OpenMP is used. But the original function is totally independent of MPI, so it is sufficient to change its error message.

Reply:
you are right, they are the same, I wanted to have two different functions, in case we want to change the strategy for OpenMP thread decomposition.

Reply:
done